rllib/utils/actor_manager.py

```diff
@@ -543,18 +567,18 @@ def foreach_actor_async(
             )
 
         num_calls_to_make: Dict[int, int] = defaultdict(lambda: 0)
-        # Drop calls to actors that are too busy.
+        # Drop calls to actors that are too busy for this specific tag.
         if isinstance(func, list):
             assert len(func) == len(remote_actor_ids)
             limited_func = []
             limited_kwargs = []
             limited_remote_actor_ids = []
             for i, (f, raid) in enumerate(zip(func, remote_actor_ids)):
-                num_outstanding_reqs = self._remote_actor_states[
+                num_outstanding_reqs_for_tag = self._remote_actor_states[
                     raid
-                ].num_in_flight_async_requests
+                ].get_num_in_flight_requests(tag)
                 if (
-                    num_outstanding_reqs + num_calls_to_make[raid]
+                    num_outstanding_reqs_for_tag + num_calls_to_make[raid]
```

Comment:
Does this mean that we have now more outstanding calls per actor because each tag has now `_max_remote_requests_in_flight_per_actor`?

Reply:
Indeed.
Maximum total number of in flight requests is now the number of tags in use times the number maximum number of in flight requests as per the config.